spec/optimizely_factory_spec.rb

```diff
@@ -30,22 +30,37 @@
   let(:event_dispatcher) { Optimizely::EventDispatcher.new }
   let(:notification_center) { Optimizely::NotificationCenter.new(spy_logger, error_handler) }
 
-  describe '.default_instance' do
-    it 'should take http config manager' do
-      allow(Optimizely::HTTPProjectConfigManager).to receive(:new)
-
-      http_project_config_manager = Optimizely::HTTPProjectConfigManager.new(
-        sdk_key: 'sdk_key',
-        datafile: datafile,
-        logger: spy_logger,
-        error_handler: error_handler,
-        skip_json_validation: false,
-        notification_center: notification_center
+  before(:example) do
+    WebMock.allow_net_connect!
+    stub_request(:get, 'https://cdn.optimizely.com/datafiles/sdk_key.json')
+      .with(
+        headers: {
+          'Content-Type' => 'application/json'
+        }
       )
+      .to_return(status: 200, body: '', headers: {})
+  end
 
+  describe '.default_instance' do
+    it 'should create http config manager when sdk is given' do
       optimizely_instance = Optimizely::OptimizelyFactory.default_instance('sdk_key', datafile)
+      expect(optimizely_instance.config_manager). to be_instance_of(Optimizely::HTTPProjectConfigManager)
+    end
 
-      expect(optimizely_instance.config_manager). to eq(http_project_config_manager)
+    it 'should create http config manager when polling interval and blocking timeout are set' do
```

Comment:
Please add in a comment, verifying polling interval and blocking timeouts are returned from config_manager when set using optimizely factory.